Sources/RichTextKit/Coordinator/RichTextCoordinator+ColorSubscriptions.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  RichTextCoordinator+ColorSubscriptions.swift
+//
+//
+//  Created by Dominik Bucher on 18.01.2024.
+//
+
+#if iOS || macOS || os(tvOS)
+import Foundation
+
+extension RichTextCoordinator {
+    
+    func setColor(_ color: ColorRepresentable, for coloredAttribute: RichTextColor) {
+        var applyRange: NSRange? = nil
+        if textView.hasSelectedRange {
+            applyRange = textView.selectedRange
+        }
+        switch coloredAttribute {
+        case .foreground:
```

Comment:
I think the function should be called `setColor(_ type: RichTextColor, to value: ColorRepresentable)` so it reads

`setColor(.foreground, to: .black)` 

Instead of

`setColor(.black, for: .foreground)` 

The current suggestion feels backwards